src/SignalR/clients/ts/signalr/src/LongPollingTransport.ts

```diff
@@ -60,7 +63,7 @@ export class LongPollingTransport implements ITransport {
             throw new Error("Binary protocols over XmlHttpRequest not implementing advanced features are not supported.");
         }
 
-        const headers = {};
+        const headers = this.headers;
```

Comment:
We want to apply user headers last so they override headers set by the library

Reply:
Fixed: 4726bcb83af1df7e0f13a419614747591925d8a0